examples/image_captioning/image_captions.txt

```diff
@@ -0,0 +1,52 @@
+image34.jpg	a duck begging for food from two women	a woman and her friend watching a duck	a duck lifting its head to two women to get fed	two women feeding a duck	women in winter jackets smiling while giving a duck food

```

Comment:
same as above. let's don't have duplicates and keep the ones on the tests/datasources

Reply:
maybe we can add something to the notebook indicating that - e.g., find those images under tests/datasources